packaging/common/rabbitmq-server-ha.ocf

```diff
@@ -0,0 +1,1585 @@
+#!/bin/bash
```

Comment:
@bogdando I'd focus on the platforms/shells you plan using this script on. We are not pedantic. That said, @dumbbell can provide you a lot of feedback on *BSD compatibility.